sicp/main.rkt

```diff
@@ -2,41 +2,67 @@
 
 (require racket/provide
          (prefix-in r5rs: r5rs)
-         (rename-in racket [random racket:random]))
+         (only-in r5rs [lambda r5rs:λ])
+         (only-in racket/base [random racket:random]))
```

Comment:
What's the reason for using lambda from R5RS and not Racket?

Reply:
`#lang sicp` seems to mainly use `r5rs`'s operators, so I guess using `r5rs`'s λ might be more consistent.